### PR TITLE
Add arguments for configuring verbosity to the bazel runner

### LINKED ISSUE #14: Add configuration for verbosity
By default, local builds should be less verbose and kokoro builds should be as verbose as possible. This behavior should be configurable.

### PR DESCRIPTION
**Pull request by jverkoey**
Closes #14

### CODE REVIEW

File: bazel.sh [hunk 1]
```diff
@@ -19,21 +19,22 @@
...
-# Arguments:
-#   1. bazel action (build or test, usually)
-#   2. BUILD target.
-#   3. Minimum Xcode version. E.g. "8" or "8.2.1"
+# Ordered arguments:
+#   1. action:  bazel ACTION. E.g. "build" or "test".
+#   2. target:  Bazel build target. E.g. "//path/to/target:Target"
+#
+# Named arguments:
+#   -m|--min-xcode-version <version>: Minimum Xcode version. E.g. "8" or "8.2.1"
```
Comment (romoore) [inline bazel.sh]: If I have three Xcodes installed:
* Xcode 8.2.0
* Xcode 8.3.3
* Xcode 9.0

And I specify a minimum version of "8.3.3", will it execute with either 8.3.3 or 9.0? With both?

File: bazel.sh [hunk 2]
```diff
@@ -19,21 +19,22 @@
...
+#   bazel.sh build //:CatalogByConvention --xcode-version 8.2
+#   bazel.sh test //:CatalogByConventionTests -v
 
 # Fail on any error.
 set -e
 
-# Display commands to stderr.
-set -x
-
 script_version="v3.2.0"
```
Comment (romoore) [inline bazel.sh:40]: Bump script version? It seems we're making breaking changes.

File: bazel.sh [hunk 3]
```diff
@@ -19,21 +19,22 @@
...
 echo "bazel_build_and_test version $script_version"
```
Comment (romoore) [inline bazel.sh]: Change this to echo "$0 version $script_version"? Or maybe something like

```bash
echo "$(basename $0) version $script_version"
```

File: bazel.sh [hunk 4]
```diff
@@ -46,38 +47,72 @@ version_as_number() {
...
+if [ -n "$VERBOSE_OUTPUT" ]; then
+  verbosity_flags="-s"
+
+  # Display commands to stderr.
+  set -x
 fi
 
+ACTION="$1"
+TARGET="$2"
+
```
Comment (romoore) [inline bazel.sh:91]: To maintain backward compatibility: If `min-xcode-version` unset, set it as `"$(version_as_number $3)"` ?  However, I'm OK with a breaking change if we bump the major version number of the script.

**Review by romoore — APPROVED** [review]
Please see comments.

File: bazel.sh (hunk 1 above)
Comment (jverkoey) [reply bazel.sh]: It'll execute every version of Xcode greater than or equal to the minimum xcode version.

Comment (jverkoey) [reply bazel.sh]: Added clarification to the comment.

File: bazel.sh (hunk 2 above)
Comment (jverkoey) [reply bazel.sh:40]: This'll be bumped as part of the release process.

File: bazel.sh (hunk 3 above)
Comment (jverkoey) [reply bazel.sh]: Done.

File: bazel.sh (hunk 4 above)
Comment (jverkoey) [reply bazel.sh:91]: Will be a breaking change.

**Pull request merged**